scripts/tools/nrfconnect/generate_nrfconnect_chip_factory_data.py

```diff
@@ -61,6 +64,82 @@ def get_raw_private_key_der(der_file: str, password: str):
         return None
 
 
+def gen_test_certs(chip_cert_exe: str, output: str, vendor_id: int, product_id: int, device_name: str):
+    CD_PATH = MATTER_ROOT + "/credentials/test/certification-declaration/Chip-Test-CD-Signing-Cert.pem"
+    CD_KEY_PATH = MATTER_ROOT + "/credentials/test/certification-declaration/Chip-Test-CD-Signing-Key.pem"
+    PAA_PATH = MATTER_ROOT + "/credentials/test/attestation/Chip-Test-PAA-NoVID-Cert.pem"
+    PAA_KEY_PATH = MATTER_ROOT + "/credentials/test/attestation/Chip-Test-PAA-NoVID-Key.pem"
+
+    attestation_certs = namedtuple("attestation_certs", ["dac_cert", "dac_key", "pai_cert"])
+
+    log.info("Generating new certificates using chip-cert...")
+
+    # generate Certifiation declaration
```

Comment:
I will add this as an optional parameter because we'll need it stored on our devices. This part of the code will only generate a CD.der file and save it to an output directory without including it in the factory data.